src/ariel-os-debug-log/src/lib.rs

```diff
@@ -117,6 +117,19 @@ mod log_macros {
             }
         }};
     }
+
+    /// Prints a line of test.
```

Comment:
That's the doc comment we have in `ariel-os-debug`:
```suggestion
    /// Prints to the debug output, with a newline.
```